### PR TITLE
docs for applicative

### PR DESCRIPTION
**Pull request by stew**

### CODE REVIEW

**Comment by rossabaker** [comment]
:+1:

**Comment by non** [comment]
:+1:

File: docs/src/main/tut/applicative.md [hunk 1]
```diff
@@ -7,3 +7,33 @@ scaladoc: "#cats.Applicative"
...
+```tut
+import cats._
+import cats.std.all._
+
+Applicative[Option].pure(1)
+Applicative[List].pure(1)
+```
+
+Like [Functor](functor.html) and [Apply](apply.html), Applicative
+functors also composes naturally with other Applicative functors, when
```
Comment (ceedubs) [inline docs/src/main/tut/applicative.md]: Minor nitpick, but I think this sentence/paragraph would read more easily if you changed this from `, when` to `. When`.

:+1: thanks for all the great docs!

**Comment by ceedubs** [comment]
Oh it's already been merged :). I'll make a PR with my suggested change.

**Comment by non** [comment]
Sorry. ;)

**Pull request merged**